docs/api/combineManagers.md

```diff
@@ -0,0 +1,26 @@
+# `combineManagers(...managers)`
```

Comment:
Voted, but fyi it looks like the poll allows you to vote multiple times — was that intentional?